pages/index.tsx

```diff
@@ -1,83 +1,88 @@
-import { ConnectButton } from '@rainbow-me/rainbowkit';
-import type { NextPage } from 'next';
-import Head from 'next/head';
-import styles from '../styles/Home.module.css';
+import { Button } from "@chakra-ui/button";
+import { Input, InputGroup, InputRightElement } from "@chakra-ui/input";
+import { Box, Flex, Heading, Text } from "@chakra-ui/layout";
+import { Select } from "@chakra-ui/select";
+import {
+  FormControl,
+  FormLabel,
+  FormErrorMessage,
+  FormHelperText,
+} from "@chakra-ui/form-control";
+import type { NextPage } from "next";
+import Head from "next/head";
+
+const exampleTokenList = [
```

Comment:
Using a TODO makes it easier to find that this need to be changed, and minimizes that diff since var names won't need to be changed. Alternatively, track this in an issue?
```suggestion
// TODO this is an example list, update token list with real list of supported tokens
const tokenList = [
```